finder-app/Makefile

```diff
@@ -0,0 +1,25 @@
+# Makefile to run writer.c
+# Author: Shweta Prasad
+
+CC := gcc
+
+# Setting up cross-complier
+ifdef CROSS_COMPILE
+CC := $(CROSS_COMPILE)gcc
```

Comment:
@shpr8875 you can simplify to `CC := $(CROSS_COMPILE)gcc` since if CROSS_COMPILE is not defined it will result in `CC := gcc`